src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -132,14 +128,193 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
+        /// <param name="isPublic">Optional. Defaults to true. If true the playlist will be included in user’s public playlists, if false it will remain private.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/follow-playlist/
+        /// </remarks>
+        public async Task FollowPlaylist(
+            string playlistId,
+            bool isPublic = true,
+            string accessToken = null
+            )
+        {
+            if (string.IsNullOrWhiteSpace(playlistId)) throw new
+                    ArgumentNullException("playlistId");
+
+            UriBuilder builder = new UriBuilder($"{BaseUrl}/playlists/{playlistId}/followers");
+            await Put(builder.Uri, isPublic, accessToken);
+        }
+        #endregion
+
+        #region GetUsersFollowedArtists
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        public async Task<PagedArtists> GetUsersFollowedArtists(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            ) => await GetUsersFollowedArtists<PagedArtists>(limit, after, accessToken);
+
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        public async Task<T> GetUsersFollowedArtists<T>(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            )
+        {
+            if (limit < 1 || limit > 50) throw new
+                    ArgumentException("A minimum of 1 and a maximum of 50 artist ids can be sent.");
+
+            UriBuilder builder = new UriBuilder($"{BaseUrl}/me/following");
+            builder.AppendToQuery("type", "artist");
+            builder.AppendToQuery("limit", limit);
+            if (!string.IsNullOrWhiteSpace(after))
+            {
+                builder.AppendToQuery("after", after);
+            }
```

Comment:
There is an extension method for this:

    builder.AppendToQueryIfValueNotNullOrWhiteSpace("after", after);